src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -1599,4 +1619,9 @@ public class TypeWithPropertiesNestingItsGenericTypeArgument<T>
         public Tuple<int, int?, Tuple<T?>>? Deep4 { get; set; }
         public Tuple<int, int, Tuple<T, int>?>? Deep5 { get; set; }
     }
+
+    public class GenericTypeWithNullableCtor<T>
```

Comment:
Nit: nothing "nullable" about the type per se:
```suggestion
    public class GenericTypeWithCtor<T>
```